src/index.ts

```diff
@@ -38,6 +38,14 @@ export { defaultMemoize, defaultEqualityCheck }
 
 export type { DefaultMemoizeOptions }
 
+type StabilityCheck = boolean | 'once'
```

Comment:
:hand: How about we just change this to an enum?

`'always' | 'once' | 'never'`